core/src/main/java/org/apache/accumulo/core/client/rfile/RFile.java

```diff
@@ -171,7 +171,8 @@
 
     /**
      * Construct the {@link Scanner} with iterators specified in a tables properties. Properties for
-     * a table can be obtained by calling {@link TableOperations#getProperties(String)}
+     * a table can be obtained by calling {@link TableOperations#getProperties(String)}. Other
+     * properties are allowed as well in the props parameter.
```

Comment:
What do you mean by `Other properties are allowed`?

Reply:
From my comment on #609 ...
> The impl of RFile.withTableProperties() will just take whatever key values are passed and put them in a map. It is almost as if this method should be renamed to "withProperties" since we allow more than just table properties. Since it is already released as public API maybe we can just elaborate more in the javadoc.

Reply:
It would be nice to expand on what `Other properties` means in the javadoc.  Below is my attempt to explain it :

Any property that impacts file behavior regardless of whether it has the `table.` prefix may be accepted and used .  For example cache and crypto properties could be passed here.